docs/README.md

```diff
@@ -167,40 +168,41 @@ To apply CSS customization in this and other blocks, follow the instructions giv
 
 | CSS Handles |
 | ----------- |
-| `installmentValue` |
+| `installments` |
+| `installmentsListContainer` |
```

Comment:
This is the only change of this list, the rest of the changes was just me ordering the list